ipaserver/install/certs.py

```diff
@@ -667,6 +690,48 @@ def is_ipa_issued_cert(self, api, nickname):
 
         return is_ipa_issued_cert(api, cert)
 
+    def disable_system_trust(self):
```

Comment:
This was only used for HTTPD NSS database and is no longer used. You can safely remove this method.